webpack.l10n.config.babel.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-console */
+import fs from 'fs';
+import config from 'config';
+import chalk from 'chalk';
+
+import webpackConfig from './webpack.prod.config.babel';
+
+const appName = config.get('appName');
+
+if (!appName) {
+  console.log(
+    chalk.red('Please specify the appName with NODE_APP_INSTANCE'));
+  process.exit(1);
+}
+
+if (process.env.NODE_ENV !== 'production') {
+  console.log(
+    chalk.red('This should be run with NODE_ENV="production"'));
+  process.exit(1);
+}
+
+const babelrc = fs.readFileSync('./.babelrc');
+const babelrcObject = JSON.parse(babelrc);
+const babelPlugins = babelrcObject.plugins || [];
+
+// Create UTC creation date in the correct format.
+const potCreationDate = new Date().toISOString()
+  .replace('T', ' ')
+  .replace(/:\d{2}.\d{3}Z/, '+0000');
```

Comment:
This is the format expected in the pot file, I borrowed this from i18n-abide. Looks to match-up to the format we are using the the django pot files and it should be in UTC hence the +0000 is correct.